doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -172,52 +290,125 @@ This resource allows you to obtain a list of subresources published by the API:
+            "temperature": 22.5
+        }
+
+## Partial entity retrieval [/v2/entities/{entityId}{?attrs}]
+
+### One entity attribute [GET /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrs: temperature (optional, string) - Attribute to be retrieved
+
++ Response 200 (application/json)
+
+        {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": {
+              "value": 22.7,
+              "type": "urn:ogc:def:phenomenon:temperature",
+              "metadata": { }
+            }
+        }
+
+### One entity attribute. Not present [GET /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrs: mistake (optional, string) - Attribute to be retrieved
+
++ Response 200 (application/json)
+
+        {
+           "type": "Room",
+            "id": "Boe_Idearium"
+        }
+
+### Several entity attributes [GET /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: 'Bcn_Welt' (required, string) - Entity ID
+    + attrs: temperature, humidity (optional, string) - Attributes to be retrieved. Coma separated list.
+
++ Response 200 (application/json)
+
+        {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": {
+              "value": 22.7,
+              "type": "urn:ogc:def:phenomenon:temperature",
+              "metadata": { }
+            },
+            "humidity": {
+              "value": 60,
+              "type": "urn:x-ogc:def:phenomenon:OGC:Humidity",
+              "metadata": { }
+            }
+        }
+            
+# Group Query Context
+
+## Query Entities [/v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
+
+### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
 
 + Parameters
     + q: temperature==22.5;humidity==60 (optional, string) - General query parameter that allows
     to filter by attribute content
     + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates. Comma separated values.
     The number of values will depend on the 'geometry' attribute.
-    + geometry: circle;radius=10000 (optional, string) - It can be any type defined by the
-    GeoJSON spec or 'circle;radius=&lt;a radius&gt;'.
+    + geometry: circle;radius=10000 (optional, string) - Geospatial area for the query. 
     The 'exterior' modifier allows to indicate that the referred area is the outside
     of the region, for instance, 'polygon;exterior'
     + limit: 20 - Limit the number of results (optional, number) - Maximum number of results to return
     + attrs: seatsNumber (optional, string) - Attributes to be returned
-    + options: count (optional, string) - count indicates that the total number of records must be returned
+    + options: count,keyValues (optional, string) - count indicates that the total number of records must be returned
```

Comment:
keyValues has been added to the options list, but the description stills referes only to count. Maybe a neutral description wuold be better (i.e. describing options itself, without referring to count or keyValues).